src/views/Home.vue

```diff
@@ -105,8 +104,6 @@ export default {
   watch: {
     whichTab() {
       if (this.whichTab === 0) {
-        this.$router.push({ name: 'program' });
-      } else if (this.whichTab === 1) {
         this.$router.push({ name: 'timetable' });
       }
     },

```

Comment:
not sure if this would do anything anymore since whichTab will always be 1! Would it be possible to remove this completely?